docs-source/content/userguide/managing-domains/domain-lifecycle/restarting.md

```diff
@@ -3,116 +3,123 @@ title: "Restarting"
-For a domain home on PV, the type of restart needed to apply the changes depends on the nature of the WebLogic configuration change:
-    * Changes to parts of the WebLogic configuration that the operator introspects, require a full restart, even if the changes are dynamic.
-      The following are the types of changes to the WebLogic Server configuration that the operator introspects:
-        * Adding or removing a cluster, server, dynamic server, or network access point
+
+For Domain in PV, the type of restart needed depends on the nature of the WebLogic domain configuration change:
+    * With operator version 3.0.0, domain configuration changes that add clusters, either configured or dynamic, cluster member servers, or standalone servers can now be performed dynamically. This support requires that the new clusters or servers are added to the domain configuration and then [triggering the operator's introspection]({{< relref "/userguide/managing-domains/domain-lifecycle/introspection.md" >}}) of that new configuration.
+    * Other changes to parts of the domain configuration that the operator introspects, require a full restart, even if the changes are dynamic.
+      The following are the other types of changes to the domain configuration that the operator introspects:
+        * Adding or removing a network access point
         * Changing a cluster, server, dynamic server, or network access point name
         * Enabling or disabling the listen port, SSL port, or admin port
         * Changing any port numbers
         * Changing a network access point's public address
-    * Other dynamic WebLogic configuration changes do not require a restart.  For example, a change to a server's connection timeout property
+    * Other dynamic WebLogic configuration changes do not require a restart. For example, a change to a server's connection timeout property
 is dynamic and does not require a restart.
-    * Other non-dynamic WebLogic configuration changes require either a manually initiated rolling restart or a full domain restart, depending on the nature of the change.
+    * Other non-dynamic domain configuration changes require either a manually initiated rolling restart or a full domain restart, depending on the nature of the change.
       For example, a rolling restart is applicable when changing a WebLogic Server `stuck thread timer interval` property. See [Restart all the servers in the domain]({{< relref "/userguide/managing-domains/domain-lifecycle/startup/_index.md#restart-all-the-servers-in-the-domain" >}}).
 
 
-#### Changing the custom domain configuration overrides
+#### Changing the domain configuration overrides
 
-Any change to domain configuration overrides requires a full domain restart.  This includes:
+With operator version 3.0.0, changes to domain configuration overrides can be applied dynamically or as part of a rolling restart. Previously, any change to configuration overrides required a full domain restart.
```

Comment:
This section leaves it unclear that some changes require rolling - it implies that all changes can be done dynamically.

Reply:
For example, non-dynamic mbean attributes can't be changed dynamically, but this section ends with `The operator then will apply the new configuration overrides dynamically or only apply the overrides when WebLogic Server instances restart, depending on the strategy that you select.`.  Which is technically correct, but doesn't let the reader know that there are caveats.